pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -291,6 +291,13 @@ void setBannedNodeIds(final List<String> values) {
           "Minimum number of peers required before starting fast sync. (default: ${DEFAULT-VALUE})")
   private final Integer fastSyncMinPeerCount = FAST_SYNC_MIN_PEER_COUNT;
 
+  @Option(
+      names = {"--fast-sync-task-cache-size"},
+      paramLabel = MANDATORY_INTEGER_FORMAT_HELP,
+      description =
+          "The size of the cache containing the tasks to be operated. (default: ${DEFAULT-VALUE})")
+  private final Integer taskCollectionCacheSize = DEFAULT_TASK_CACHE_SIZE;
```

Comment:
Can we move this option into `SynchronizerOptions` and use the "experimental" flag style (beginning with `--X`) similar to: https://github.com/PegaSysEng/pantheon/blob/12d19a152d238ea5629e20997963865aa4784ee2/pantheon/src/main/java/tech/pegasys/pantheon/cli/options/SynchronizerOptions.java#L56-L65